CorsixTH/Lua/world.lua

```diff
@@ -544,14 +554,14 @@ end
 
 function World:createEarthquake()
```

Comment:
Assuming you figured out what this function does, it would be helpful to document it